sips/minutes/_posts/2017-02-14-sip-minutes.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: sip-landing
+title: SIP Meeting Minutes - 14th February 2017
+---
+
+# Minutes
+
+The following agenda was distributed to attendees:
+
+| Topic | Reviewer |
+| --- | --- |
+| [SIP-XX: Improving binary compatibility with @stableABI](http://docs.scala-lang.org/sips/pending/binary-compatibility.html) | Dmitry Petrashko |
+| [SIP-NN - Allow referring to other arguments in default parameters](http://docs.scala-lang.org/sips/pending/refer-other-arguments-in-args.html) | Pathikrit Bhowmick |
+| [SIP 25 - @static fields and methods in Scala objects(SI-4581)](http://docs.scala-lang.org/sips/pending/static-members.html) | Dmitry Petrashko, Sébastien Doeraene and Martin Odersky |
+| [SIP-NN - Match infix & prefix types to meet expression rules](http://docs.scala-lang.org/sips/pending/make-types-behave-like-expressions.html) | Oron Port |
+
+Jorge Vicente Cantero was the Process Lead. Travis (@travissarles) was acting secretary of the meeting.
+
+## Date, Time and Location
+
+The meeting took place at 5:00pm Central European Time / 8:00am Pacific Daylight
+Time on Tuesday, 14th February 2017 via Google Hangouts.
+
+Minutes were taken by Travis Lee, acting secretary.
+
+## Attendees
+
+Attendees Present:
+
+* Martin Odersky ([@odersky](https://github.com/odersky)), EPFL
+* Dmitry Petrashko (@DarkDimius)(https://github.com/DarkDimius), EPFL
```

Comment:
The markup is wrong here. The handle should be in square brackets and the whole thing in round parens.